MinecraftClient/Settings.cs

```diff
@@ -161,6 +161,9 @@ public static byte MCSettings_Skin_All
         //Auto Eating
         public static bool AutoEat_Enabled = false;
         public static int AutoEat_hungerThreshold = 6;
+        
+        //Auto Looking
+        public static bool AutoLook_Enabled = false;
```

Comment:
Seems like a leftover from your previous Pull Request. Can be removed from Settings.cs.